src/services/http_request_service.ts

```diff
@@ -335,17 +344,29 @@ export default class HttpRequestService {
     // frustration, I filed an issue on deno about my findings; and artisonian
     // gave an example of a working copy. Great work. Thank you!
     if (contentType && contentType.includes("multipart/form-data")) {
-      let boundary: string;
+      let boundary: null|string = null;
       try {
-        boundary = contentType.match(/boundary=([^\s]+)/)[1];
+        const match = contentType.match(/boundary=([^\s]+)/);
+        if (match) {
+          boundary = match[1];
+        }
+        if (!boundary) {
+          return ret;
+        }
       } catch (error) {
         throw new Error(`Error trying to find boundary.\n` + error.stack);
       }
+      if (!boundary) {
+        return ret;
+      }
       try {
+        const maxMemory = options && options.memory_allocation && options.memory_allocation.multipart_form_data
```

Comment:
We’re checking twice to set 10MB? I saw this up above. Should come back around to this to see how to clean it up